dsls/sql/src/main/java/org/apache/beam/dsls/sql/example/BeamSqlExample.java

```diff
@@ -58,16 +60,31 @@ public static void main(String[] args) throws Exception {
     PCollection<BeamSQLRow> inputTable = PBegin.in(p).apply(Create.of(row)
         .withCoder(new BeamSqlRowCoder(type)));
 
-    //run a simple SQL query over input PCollection;
-    String sql = "select c2, c3 from TABLE_A where c1=1";
-    PCollection<BeamSQLRow> outputStream = inputTable.apply(BeamSql.simpleQuery(sql));
+    //Case 1. run a simple SQL query over input PCollection with BeamSql.simpleQuery;
+    PCollection<BeamSQLRow> outputStream = inputTable.apply(
+        BeamSql.simpleQuery("select c2, c3 from TABLE_A where c1=1"));
```

Comment:
Agree, a global schema namespace doesn't sound good in DSL, would address the issue in a new task.